tests/libwallet_api_tests/main.cpp

```diff
@@ -984,17 +1010,87 @@ TEST_F(WalletTest2, WalletCallbackNewBlock)
 
 }
 
+TEST_F(WalletManagerMainnetTest, CreateOpenAndRefreshWalletMainNetSync)
+{
+
+    Bitmonero::Wallet * wallet = wmgr->createWallet(WALLET_NAME_MAINNET, "", WALLET_LANG);
+    MyWalletListener * wallet_listener = new MyWalletListener(wallet);
```

Comment:
Yep, but I guess for the purpose of test it is ok? Anyway, wrapped it into smart pointer